### DISCUSSION TITLE
Elastic pipeline scripts

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#8170] [Unsupported Versions] [answered]
Hi!

For my ingest pipelines I need to use scripts sometimes. I can add it manually on every affected node (via API) and then edit ingest on manager.
I'd like to know is there more convenient way to add them? With salt for example?

**Comment by weslambert** [comment, marked as answer]
You could certainly try using Salt, but we currently don't manage custom scripts for Elastic. You might try adding them to a custom template, or inline with the pipeline config if able.
